spotipy/client.py

```diff
@@ -330,6 +331,14 @@ def user(self, user):
         '''
         return self._get('users/' + user)
 
+    def me_playlists(self, limit=50, offset=0):
```

Comment:
I'm going to change this to 'current_user_playlists' to match other endpoints that work on the current user.